CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+# [v1.0.0-beta.1](https://github.com/kubermatic/kubeone/releases/tag/v1.0.0-beta.1) - 2020-07-02
+
+## Known Issues
+
+* machine-controller is failing to sign Kubelet CertificateSingingRequests (CSRs) for worker nodes due to [missing hostname in the Machine object](https://github.com/kubermatic/machine-controller/issues/781). We are currently working on a fix. In meanwhile, you can sign the CSRs manually by following [instructions](https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/#approving-certificate-signing-requests) from the Kubernetes docs.
```

Comment:
Probably this like should include and emphasize that this problem is only about GCE.

Reply:
Done, PTAL now.